### PR TITLE
Bugfix for #107

### PR DESCRIPTION
**Pull request by HuanhuanSunMSFT**
@EricMSFT @AndreSayreMSFT @Microsoft/poshtools

### CODE REVIEW

File: PowerShellTools/Intellisense/IntelliSenseManager.cs [hunk 1]
```diff
@@ -305,29 +310,24 @@ private static string ToCommandName(Guid commandGroup, uint commandId)
...
+            catch (Exception ex)
+            {
+                Log.Warn("Failed to start IntelliSense", ex);
+            }
         }
 
         private void StartIntelliSense(int lineStartPosition, int caretPosition, string lineTextUpToCaret)
         {
-            Interlocked.Increment(ref _triggerTag);
+            long triggerTime = DateTime.UtcNow.Ticks;
```
Comment (HuanhuanSunMSFT) [inline PowerShellTools/Intellisense/IntelliSenseManager.cs]: I didn't make this change. This should be caused by merge. Which one should we keep? @EricMSFT

Comment (ASayre) [inline PowerShellTools/Intellisense/IntelliSenseManager.cs]: We should be keeping the references to the Interlocked, incremented long

Comment (ericdevg) [inline PowerShellTools/Intellisense/IntelliSenseManager.cs]: Yes, we should keep incremental long.

**Comment by ASayre** [comment]
looks good!

**Pull request merged**